backend/packages/Upgrade/src/api/models/Segment.ts

```diff
@@ -37,6 +37,9 @@ export class Segment extends BaseModel {
   })
   public type: SEGMENT_TYPE;
 
+  @Column('text', { array: true, nullable: true })
+  public tags: string[];
```

Comment:
The typescript should be fixed here
`public tags?: string[]`

There are other parameters where this problem already exists. We should fix this in a separate PR, I guess.